src/main/java/it/geosolutions/httpproxy/HTTPProxy.java

```diff
@@ -199,6 +187,40 @@ HttpHost getHost(String proxyHostKey, String proxyPortKey) {
         return httpHost;
     }
 
+    /**
+     * Returns the HttpRoutePlanner based on the target host http scheme
+     * @param httpHost
+     * @param httpsHost
+     * @return HttpRoutePlanner
+     */
```

Comment:
These parameters Javadoc don't tell us anything useful, they are just noise, please remove them.

This Javadoc doesn't tell us much:
```
Returns the HttpRoutePlanner based on the target host http scheme
```